fuzz/fuzz_targets/common.rs

```diff
@@ -0,0 +1,75 @@
+use boa_ast::{
+    visitor::{VisitWith, VisitorMut},
+    Expression, StatementList,
+};
+use boa_engine::context::Context;
+use boa_interner::Sym;
+use libfuzzer_sys::arbitrary;
+use libfuzzer_sys::arbitrary::{Arbitrary, Unstructured};
+use std::fmt::{Debug, Formatter};
+use std::ops::ControlFlow;
+
+/// Context for performing fuzzing. This structure contains both the generated AST as well as the
+/// context used to resolve the symbols therein.
+pub struct FuzzData {
+    pub context: Context,
```

Comment:
This should be changed to an `Interner` field. The context is unused in both the parser fuzzer and the VM fuzzer (you can pass to `ContextBuilder` an `Interner`).

Reply:
The VM fuzzer needs the context to track the instructions remaining.

Reply:
Yeah, and you can create a new `Context` inside `do_fuzz` and pass it the interner of `FuzzData`, IIUC.

Reply:
Aha, got it.